### ISSUE TITLE
Slick in looping mode makes website completely unusable for screenreader users

### ISSUE THREAD

**Issue by tamaracha** [kenwheeler/slick#2749]
When Slick is used in infinite looping mode, screenreaders anounce any change, which is very disrupting and annoying for the whole website UX. This happens at your demo GH page and I encountered this problem at www.allyouneed-fresh.de. The latter one is an online supermarket and Slick prevents me from using it properly. They use Slick for their ads and the screenreaders read out their nonsense in the items instead of letting me read the main page content. It has become worse than a real supermarket. So I'm not using Slick myself in development, but wanted to find out the roots of the problem with this shop. Unfortunately, there is no more comparable screenreader accessible online supermarket in Germany which I could use instead.

I tried Voiceover with Safari on Mac OS X and NVDA with Firefox on Windows 10. Both screenreaders have difficulties with the fast rotation rate.

In my opinion, it is not appropriate to make Slick a live region, although it updates its content. Statistically, auto-looping carusels are used for ornamental purposes (for visual fun) and often display peripheral, non-central content. Blind screenreader users have to explore the page structure actively in a more sequential way, this process can be demanding and cognitive capacities are limited. So distraction becomes annoying quickly. . Live regions, even polite ones, should be used carefully to anounce content which is essential for the user to interact with the app or site. So it might be preferable to remove the aria-live attribute at least in looping mode, and keep those accessibility features which allow manual rotation.

**Comment by leggomuhgreggo** [comment]
It's true--top of the list for next version. There're several issues about this so I'm gonna close this one but I put the a11y label and I will review it with the others when we're coming up with a solution.

Thanks
